packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/create/iti39-envelope.ts

```diff
@@ -58,8 +58,8 @@ export function createITI39SoapEnvelope({
       "@_xmlns:urn": namespaces.urnihe,
       "urn:RetrieveDocumentSetRequest": {
         "urn:DocumentRequest": documentReferences.map(docRef => ({
-          "urn:HomeCommunityId": docRef.homeCommunityId,
-          "urn:RepositoryUniqueId": docRef.repositoryUniqueId,
+          "urn:HomeCommunityId": wrapIdInUrnOid(docRef.homeCommunityId),
```

Comment:
this is resolving a bug when querying epic endpoints where they require these ids to be wrapped

Reply:
I expected this function to come from `shared`, but NBAD